### ISSUE TITLE
Some emails contain the same starting alpha-numeric characters and autocomplete incorrectly...

### ISSUE THREAD

**Issue by bjornmeansbear** [cape-io/mica.ezle.io#15]
for instance, jhudson01@mica.edu contains jhu@mica.edu
so, when starting to type j-h-u- for jhudson, the system just assumes you mean jhu and auto-completes, sending the email to the wrong person... need to investigate a solution to this for AY2016

**Comment by webmasterkai** [comment]
The simple solution is to use full email addresses.

**Comment by bjornmeansbear** [comment]
Or we could wait until they input the @, but yes, full emails will solve it.... It was too simple for me to think of

Please pardon brevity and mistakes

**Comment by webmasterkai** [comment]
True. I'm going for full emails since it'll be more universal. We can use the same login component on all our sites/apps.

**Comment by bjornmeansbear** [comment]
Perfect! Modular solutions!

Please pardon brevity and mistakes
